### PR TITLE
vignettes and make_quarto_rum()

### PR DESCRIPTION
**Pull request by kyleGrealis**
Please review the files listed below. I have completed the first round of the vignettes for review and edits. 

Also, I was able to recreate the make_project.R file as make_quarto_project.R successfully. I then ran a build check that ~~produced 2 notes (please refer to the picture included in the email I will be sending immediately after this), though they weren't substantial enough to create errors or warnings~~ (this has since been fixed) did not produce errors. Next, I used `devtools::load_all(".") to then run `make_quarto_project(getwd())`. The boilerplate version is produced with necessary Quarto syntax changes and will knit as expected.

Again, please provide any feedback as this is my first time (1) creating a vignette and (2) forking a project to add new features.

Thank you for allowing me to participate.

### CODE REVIEW

File: vignettes/make_research_project.Rmd [hunk 1]
```diff
@@ -10,23 +10,170 @@ vignette: >
...
+    
+1. Run the following script in your console:
+
+```
+PATH <- getwd()
+make_project(PATH)
+```
+
+
+Let the `rUM` soak in and easy your worries.
```
Comment (gabrielodom) [inline vignettes/make_research_project.Rmd:78]: "ease" your worries

**Review by gabrielodom — COMMENTED** [review]
Are you 100% sure that you can't make this code work without the full @import call? @RaymondBalise, some of these package calls can be moved to Suggests, right?

**Comment by gabrielodom** [comment]
@kyleGrealis, please send me an email. We wills schedule a zoom call to figure it out.

**Comment by kyleGrealis** [comment]
removing pull request from the forked version. the other files have been added after the discussed changes to the `make_project()` function that enables options for quarto or markdown

**Pull request closed without merging**